### PR TITLE
feat: added a standard placement of price on events

### LINKED ISSUE #9149: [FEATURE] Improvement - standard placement of price on events
### Description

On the events page, currently the location of the price is dependent upon the length of text due to the placement inside the div.  This creates a difficult to visually navigate situation at the price is not always in the same location on each event.

This issue is to create a standard location for the event price in the event cards so that it is no longer dependent on the description length and is easier to visually navigate.

See screenshot for an example of the issue.

### Screenshots

events-price

### Do you want to work on this issue?

No

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by AbhijeetBiswal**
Fixes #9149

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
new event price

## Note to reviewers
added a standard placement to the price on events

### CODE REVIEW

File: components/event/EventCard.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ export default function EventCard({ manage, event, usernames }) {
                   </span>
                 </div>
                 {event.price?.startingFrom > 0 && (
-                  <div>${event.price?.startingFrom}</div>
+                  <div className="ml-auto mr-[-35px]" >${event.price?.startingFrom}</div>
```
Comment (eddiejaoude) [inline components/event/EventCard.js:105]: Can this be done with flex? px is very specific and fragile

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution. I have added an inline comment

File: components/event/EventCard.js (hunk 1 above)
Comment (AbhijeetBiswal) [reply components/event/EventCard.js:105]: i have added the mr-[-35px] to make the price text align on the bottom of the profile picture.

Comment (eddiejaoude) [reply components/event/EventCard.js:105]: Using flex would be better, being that specific will break when something else changes

Comment (Aladdin4u) [reply components/event/EventCard.js:105]: > i have added the mr-[-35px] to make the price text align on the bottom of the profile picture.

I think align-self will work better in this context

**Comment by AbhijeetBiswal** [comment]
@eddiejaoude I have added some new changes

**Comment by eddiejaoude** [comment]
> @eddiejaoude I have added some new changes

Thank you, I will take a look later today. Can you make sure the screenshot is updated also.

**Comment by Himanshu-0981** [comment]
This still causes problems. When the content size is smaller, the price tag still changes its position. It can be solved just by changing the element positions.

**Comment by AbhijeetBiswal** [comment]
I have fixed this particular issue in this PR

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**